GAE/src/com/gallatinsystems/surveyal/dao/SurveyedLocaleDao.java

```diff
@@ -40,6 +40,8 @@
  */
 public class SurveyedLocaleDao extends BaseDAO<SurveyedLocale> {
 
+    private static final String LT_OP = "<";
```

Comment:
What do you think if we move this constant to `BaseDAO` similar to the rest of them?

Reply:
Sure